app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'leshoyasoap'
+require 'colorize'
+# Ask for a username
+
+
+a =  Dir.glob("/Users/leshoyastarkey/Dropbox/SoapBox/*")
+
+a. each do |element|
+
+  file  = File.open(element)
+
+
+  answers = {username: file.read,
+  body: File.basename(element),
+  created_at: file.birthtime}
+
+  puts answers[:username]
+  puts answers[:body]
+  puts answers[:created_at]
+  puts "-" * 50
+end
+
+#Asks for My Username + saves the information in a local file
+puts "What is your username?".colorize(:blue)
+  usercommand =  gets.chomp.capitalize
+    file2 = File.open("/Users/leshoyastarkey/work/soapbox/soapboxuser", "w")
```

Comment:
Super nit-picky, but lines 25-27 don't need to be indented because they are not within a block like `if` or `do`